### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by Guekka** [BrenekH/encodarr#149] [General]
Hey,
I like the looks of this app, but it looks like there was no real activity since the end of 2021.
Is encodarr ready for production? Has the project been abandoned?

Whatever the answer is, it will be alright, I know how hard it is to develop an open source project. I'm just trying to see if I can use it :slightly_smiling_face:

**Comment by BrenekH** [comment]
Sorry about the delayed response. I saw this yesterday but kinda forgot to respond 🙈.

> Is encodarr ready for production?

It really depends. For my specific use case of converting videos to HEVC and down sampling the 1st audio track to stereo, I would call it production adequate (there's some things relating to displaying FFmpeg errors that I never got around to that would be desirable for "production"). If you're wanting to convert to some other format(H.264 perhaps?), it will probably work fine, but I can't guarantee that with 100% certainty.

> Has the project been abandoned?

Kind of? As previously mentioned, the use case that I first had in mind is essentially feature complete. As time went on, the scope in my mind grew to include features that would expand the capabilities such as a plugin system similar to Tdarr's and some scheduling features, but then I started my computer science degree and lost both time and motivation to continue working on it. Encodarr still pops into my mind about once a week as something I would like to get back to, but the reality is that I don't have a reason to come back to it except for my own self-satisfaction.

If you do end up trying Encodarr, I would still love to hear any feedback you may have, even if you don't end up using Encodarr as a long-term solution. Who knows? I may one day have a reason to return to the project and "finish" it off.

**Reply by Guekka** [reply]
Haha, "one day" is not a delayed response. You don't have to apologize

Alright, I get it. Thanks for the answer
I don't think I will use it (or Tdarr) because it seems too complicated for my needs. I basically just need to convert my old library
